src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.HtmlImportMap.targets

```diff
@@ -0,0 +1,81 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.StaticWebAssets.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+    
+  <UsingTask TaskName="Microsoft.AspNetCore.StaticWebAssets.Tasks.WriteImportMapToHtml" AssemblyFile="$(StaticWebAssetsSdkBuildTasksAssembly)" />
+
+    <PropertyGroup>
+    <ResolveBuildRelatedStaticWebAssetsDependsOn>
+      $(ResolveStaticWebAssetsInputsDependsOn);
+      _UpdateHtmlFiles;
```

Comment:
Note: This needs to happen also before the service-worker-manifest gets generated.

Reply:
Why is that? The html documents are not part of service worker assets manifest, aren't they?